### ISSUE TITLE
Sonoff Pow Support

### ISSUE THREAD

**Issue by epiller** [KmanOz/KmanSonoff#14]
Could the support for Sonoff Pow be added? 
It’s just like the Sonoff Basic, with an additional read-only topic that reports the wattage.

**Comment by disrupted** [comment]
+1 would really appreciate that as I prefer the stability of this firmware to the complexity of something like Tasmota

**Comment by paulscode** [comment]
I would love to use this for my crypto mining rigs -- remote control and monitoring power usage, and handles up to 16A.  All of the other supported devices (except for TH16) have a max of 10A or less.

**Comment by ystrem** [comment]
Hi, what's needed to implement ? The power measurement ?

**Comment by epiller** [comment]
@ystrem Yes!

**Comment by master-kenobi** [comment]
Yes. Please support Sonoff POW. I was a bit surprise that it is not supported because the older "Sonoff-HomeAssistant" version was supported.

**Comment by ppanagiotis** [comment]
Any update about this?

**Comment by pitw** [comment]
I just flashed a Sonoff POW with this lib - it works.
Just use the Single Channel Version.
If not working try to change the  Payload Config for the Switch in Homeassistant:

```
//Replace on with 1on /off with 1off
payload_on: "1on"
payload_off: "1off"
```

**Comment by bereska** [comment]
@pitw what sonoff POW do you have? are you able to read power/voltage as well? thanks
